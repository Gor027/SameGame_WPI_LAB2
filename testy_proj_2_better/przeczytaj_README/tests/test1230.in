_ 6 _6_6  666_6 __6__66__  _6 6_ 6_66__6_   66____6 ___666 _6    666  666
 _  6_6 6 6_ 6___ 66_ 6 6 _6 _ _6 6 6_ 666_ 666_ 6 _6 66____66 6666 _6 _ __6___6
6 66 6___ __ ___ 66_6_66_6 6_   6  6 666 _ _666_6  6666 _ _6_6 _66_ _  
66 66_ 6_ 6_ _6 _ 6 6__6  _6________66___  __ _ _6__ 6 _ 6 66_   ___6
   _66__ 6_6 666  __  6 6  __66 66__   _ 6 6_ _6    6 6__66 66 _66
  6_ _66_6_   6_6 ___   
  _ 6 _666_ _ _   _  _6 6   _
 6   _ __6_6 6
6__ 66_ 6__ _6_6 6_  6_ _ ___6 _ 66 _ 6_6 __6 _666__66___   6 6  __6__ ___ __6
