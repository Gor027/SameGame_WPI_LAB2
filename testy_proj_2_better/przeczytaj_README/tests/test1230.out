          6
          6   6   6
      _66 _  __   6   6
     66__ 66 6_   6   6 _  _6          6    6 _             6    6
     __6_6__ _6_  6__66 6  6_     _    6    6 6_    _ _66   6    66
     6___6__ _6__ _6___ _  6__ _66666__6_  _6 6___ _6_66_6 __6   66 _
     _6666__6666____6_6_6 _6__6_6__66_6_66 __6__6666666_____66 66__6_  6
 ___ __6_6_6_6___6__6_666 ___6___6__66_____6_6_6_6__666__6_66_ _66__66 _6_ 6__
________6__6_6_666___6_6_6___66_666__66_66__66_666__66___666666_6__6__6______6_6
